.github/workflows/test.yml

```diff
@@ -26,15 +26,16 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        java: [ '1.8', '11' ]
+        java: [ '8', '11', '17' ]
     name: Build Parquet with JDK ${{ matrix.java }}
 
     steps:
       - uses: actions/checkout@master
       - name: Set up JDK ${{ matrix.java }}
-        uses: actions/setup-java@v1
+        uses: actions/setup-java@v3
         with:
           java-version: ${{ matrix.java }}
+          distribution: temurin
```

Comment:
@Fokko Why do we need to specify a distribution?

Reply:
This is now required, before it would just take the Oracle one IIRC